tests/test_basic_s3.py

```diff
@@ -30,7 +30,7 @@ def test_can_make_request(s3_client):
 def test_fail_proxy_request(aa_fail_proxy_config, s3_client):
     # based on test_can_make_request
 
-    with pytest.raises(ProxyConnectionError):
+    with pytest.raises(aiohttp.ClientConnectorError):
```

Comment:
During my tests, the exception thrown to the test case was `ClientConnectorError`. The docstring of `ClientConnectorError` describes its purpose as:
```
Raised in :class:`aiohttp.connector.TCPConnector`
if connection to proxy can not be established.
```

I have also tried `aiohttp.ClientProxyConnectionError` but it was raised as the part of nested exception chain and not caught by the test code.

Exception chain:
* `ConnectionRefusedError` (asyncio/selector_events.py, line 481) ->
* `aiohttp.client_exceptions.ClientConnectorError` (aiohttp/connector.py, line 678) ->
* `aiohttp.client_exceptions.ClientProxyConnectionError` (aiohttp/connector.py, line 692) ->
* `aiohttp.client_exceptions.ClientConnectorError` (aiohttp/connector.py, line 379) ->
* test code